Apps/g.Tec/g.NEEDaccess/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+cmake_minimum_required(VERSION 3.0)
+project(gNEEDaccess VERSION 0.0.1)
+
+# load LSLAppBoilerplate if not done already
+if(NOT TARGET LSL::lsl)
+    if(NOT LSL_ROOT)
+        set(LSL_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../../build/lsl_Release/lslinstall/LSL")
```

Comment:
This could be somewhere else if a different install path is used (either as an argument to `lslbuild.py`, with manual CMake invocations, with debug builds, Apps with a different subdirectory level, ...). It's convenient in this case, but I
d leave it out

Reply:
This only assumes a default location for if not building in the full tree and LSL_ROOT has not been provided. Even then, if the lib isn't found in this assumed directory, it fails saying that LSL_ROOT needs to be set when building out of tree. This shouldn't break anything for anyone and it just makes things a little easier on myself for development.

Reply:
I added some comments to make it clear that this should be ignored by everyone except me.